static/app/views/starfish/views/webServiceView/spanGroupBreakdown.tsx

```diff
@@ -45,6 +52,13 @@ export function SpanGroupBreakdown({
   const {selection} = usePageFilters();
   const theme = useTheme();
   const [showSeriesArray, setShowSeriesArray] = useState<boolean[]>(initialShowSeries);
+  const options: SelectOption<string>[] = [
```

Comment:
```suggestion
  const options: SelectOption<DataDisplayType>[] = [
```
nit, but i think you can probably use `DataDisplayType` as the type for the option state so it's not just a generic `string`